README.md

```diff
@@ -27,6 +27,9 @@ running on OpenStack.  There is a Telegraf application, cs:telegraf, collecting
 metrics from units, and we wish to relate that to Prometheus and Grafana running
 in another model named cos, running Kubernetes.
 
+If you already have a working COS Lite deployment, you can skip creating another
+one, as well as the steps where you would deploy the COS Lite components one by one.
```

Comment:
**suggestion (non blocking):** Split this one big section into several, one of which being "deploy cos", under which we'd say "skip if you already have".